docs/source/app-dev/ledger-api.rst

```diff
@@ -52,6 +52,22 @@ For full information on the services see :doc:`/app-dev/services`.
 
 You may also want to read the :doc:`protobuf documentation </app-dev/grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
 
+How to Access the Ledger API
+****************************
+
+You can access the Ledger API via the :doc:`Java Bindings <bindings-java/index>`.
+
+If you don't use a language that targets the JVM you can use gRPC to generate the code to access the Ledger API in
+several supported programming languages. :doc:`Further documentation <bindings-x-lang/index>` provides a few
+pointers on how you may want to approach this.
+
+You can also use the :doc:`HTTP JSON API Service </json-api/index>` to tap into the Ledger API.
+
+At its core, it provides a simplified view of the active contract set and additional primities to query it and
```

Comment:
```suggestion
At its core, this service provides a simplified view of the active contract set and additional primities to query it and
```

Reply:
Also, is "primities" correct here?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/13271/commits/eec658d36352eb6efe99ed501d80b68f1a12e050